event/consumer_test.go

```diff
@@ -92,6 +98,33 @@ func TestToEvent(t *testing.T) {
 	})
 }
 
+func TestClose(t *testing.T) {
+
+	Convey("Given a consumer", t, func() {
+
+		messages := make(chan kafka.Message, 1)
+		messageConsumer := kafkatest.NewMessageConsumer(messages)
+		handler := eventtest.NewEventHandler()
+
+		expectedEvent := getExampleEvent()
+
+		message := kafkatest.NewMessage(Marshal(*expectedEvent))
+		messages <- message
+
+		consumer := event.NewConsumer()
+		go consumer.Consume(messageConsumer, handler)
+
+		Convey("When close is called", func() {
+
+			err := consumer.Close(context.Background())
+
+			Convey("The expected event is sent to the handler", func() {
+				So(err, ShouldBeNil)
+			})
+		})
+	})
+}
+
 // Marshal helper method to marshal a event into a []byte
 func Marshal(event event.DimensionsInserted) []byte {
```

Comment:
Dont export helpers from tests if we can avoid it - looks like all the test files in this package share a package name so shouldn't need to export this.

Reply:
Done